Add info about README generator in CONTRIBUTING (#8052)
bitnami/charts: CONTRIBUTING.md

### Documentation Requirements

> - A chart's `README.md` must include configuration options. The tables of parameters are generatedi, based on the metadata information in the values.yaml file, by using [this tool](ihttps://github.com/bitnami-labs/readme-generator-for-helm).

Reviewer: ```suggestion
- A chart's `README.md` must include configuration options. The tables of parameters are generated based on the metadata information from the `values.yaml` file, by using [this tool](ihttps://github.com/bitnami-labs/readme-generator-for-helm).
```
